velox/functions/prestosql/tests/ArrayMaxTest.cpp

```diff
@@ -146,9 +146,17 @@ TEST_F(ArrayMaxTest, longVarcharNoNulls) {
 // Test documented example.
 TEST_F(ArrayMaxTest, docs) {
   auto input = makeNullableArrayVector<int32_t>(
-      {{1, 2, 3}, {-1, -2, -2}, {-1, -2, std::nullopt}, {}});
-  auto expected =
-      makeNullableFlatVector<int32_t>({3, -1, std::nullopt, std::nullopt});
+      {{1, 2, 3},
+       {-1, -2, -2},
+       {-1, -2, std::nullopt},
+       {},
+       {-0.0001, -0.0002, -0.0003, std::numeric_limits<double>::quiet_NaN()}});
```

Comment:
nit: can you add a case with both null and NaN in the input. Also for arrayMin()